frontend_tests/node_tests/emoji_picker.js

```diff
@@ -125,3 +132,61 @@ set_global('popovers', {
     assert($(".emoji-popover-category-tabs").visible());
     assert(!$(".emoji-search-results-container").visible());
 }());
+
+(function test_toggle_emoji_popover() {
+    var element = $('fake-add-emoji');
```

Comment:
I don't know the exact intentions here, but I think you want to use `$.create()` for all selectors that don't actually match things in the real code.